faunadb-java-dsl/src/main/java/com/faunadb/client/errors/FaunaException.java

```diff
@@ -48,12 +51,13 @@ public int status() {
     }
   }
 
-  private static String constructErrorMessage(ImmutableList<HttpResponses.QueryError> errors) {
-    ImmutableList.Builder<String> messages = ImmutableList.builder();
+  private static String constructErrorMessage(List<HttpResponses.QueryError> errors) {
+    List<String> messages = new ArrayList<>();
+
     for (HttpResponses.QueryError error : errors) {
       messages.add(error.code() + ": " + error.description());
     }
 
-    return Joiner.on(", ").join(messages.build());
+    return Joiner.on(", ").join(messages);
```

Comment:
Are you planning to remove this `Joiner` too? I think it comes from the same collections library.

Reply:
At some point, yes. I can continue with this PR or create new ones for extra guava exorcism. Do you have a preference?

Reply:
I'm find either way. I just wanted to confirm that we're removing guava for good.

Reply:
I have removed Joiner.